src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -150,8 +173,45 @@ Domain<3> BinaryCompactObject::create_domain() const noexcept {
   std::move(maps_cube_B.begin(), maps_cube_B.end(), std::back_inserter(maps));
   std::move(maps_frustums.begin(), maps_frustums.end(),
             std::back_inserter(maps));
-  std::move(maps_outer_shell.begin(), maps_outer_shell.end(),
-            std::back_inserter(maps));
+
+  if (outer_spherical_shell_layers_ > 1) {
+    // wedge_coordinate_maps() can only include more than one layer
+    // if the sphericity is constant. Therefore, we surround the outermost
+    // cubed-sphere (made as a single layer with sphericity varying from 0.0
+    // on its inner boundary to 1.0 on its outer boundary) with
+    // (outer_spherical_shell_layers_) - 1 layers each with constant
+    // sphericity == 1
+    const double inner_radius_first_outer_shell =
+        sqrt(3.0) * 0.5 * length_outer_cube_;
+    const double outer_radius_first_outer_shell =
+        use_logarithmic_map_outer_spherical_shell_
+            ? inner_radius_first_outer_shell +
```

Comment:
I am surprised by that when `use_log_map` is `true`, this does a linear division. Can you explain this to me or point me to something that explains it?

Reply:
D'oh! That's a bug. It should be (not use_logarithmic_map_outer_spherical_shell_) ? ... : ... Good catch, and thanks for the very quick review!!

Reply:
Do you know why this wasn't caught in the test? Can you add a test that would catch it?